src/Modalizer.ts

```diff
@@ -272,6 +272,10 @@ export class Modalizer
                     findPropsOut
                 );
 
+                if (next === null) {
```

Comment:
Would be great to add a test for this, you can add it here and copy code from other tests to get started https://github.com/microsoft/tabster/blob/master/tests/Modalizer.test.tsx

You can run tests with `npm run test`